dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -1319,8 +1326,10 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 						fullWidth={true}
 						data-print-layout="hide"
 						padSides={false}
-						backgroundColour={brandBackground.primary}
-						borderColour={brandBorder.primary}
+						backgroundColour={themePalette(
+							'--article-section-background',
+						)}
```

Comment:
Dont think article-section-background should be used here, looks like it should be sourcePalette.brand[400] again